section3/reflection.md

```diff
@@ -1,17 +1,22 @@
 ## Section 3 Reflection
 
 1. How different did your workflow feel this week, considering we asked you to follow the Pomodoro technique?
-
+***My workflow felt a little more interrupted with the use of the Pomodoro technique. I think with some adjustment of the intervals, it will be a useful technique.***
 2. Regarding the work you did around setting intentions in Step 1 of the Pomodoro technique - how did that go? Were you surprised by anything (did you find yourself way more focused than you realized, more distracted that you thought you'd be, estimating times accurately or totally off, etc)?
-
+***My time estimates for completing tasks were more off than I expected - it generally took me longer than I expected. I was surprised by how often I felt interrupted by my break timer.***
 3. What is a Hash, and how is it different from an Array?
-
+***A hash allows you to store a list of items and assign them to whatever you'd like, as opposed to just numbers. Arrays store lists of items but assign them to index numbers.***
 4. In the space below, create a Hash stored to a variable named `pet_store`.  This hash should hold an inventory of items and the number of that item that you might find at a pet store.
+`pet_store = {dogs: 10, cats: 11, fish: 12, hamsters: 13, rats: 14}`
 
 5. Given the following `states = {"CO" => "Colorado", "IA" => "Iowa", "OK" => "Oklahoma"}`, how would you access the value `"Iowa"`?
-
+`puts states["IA"]`
 6. With the same hash above, how would we get all the keys?  How about all the values?
-
+```
+puts states.keys
+puts states.values
+```
 7. What is another example of when we might use a hash?  In your example, why is a hash better than an array?
-
-8. What questions do you still have about hashes?
\ No newline at end of file
+***A hash could be used to store values related to someone's bank account. A hash would be be better than an array in this situation because there are multiple types of unique data that can be stored in the bank account***
+8. What questions do you still have about hashes?
+***After establishing an array and assigning values to the elements using a hash, how do you reference or list the values of the original array? Do the elements need to have values associated with them in order to be reference?***
```

Comment:
Hmmm I'm not sure I fully understand what you are asking. Could you provide an example of an array or hash that meets the criteria of what you're asking about? Happy to dig into it with you